controllers/admin/AdminLoginController.php

```diff
@@ -102,7 +102,12 @@ public function initContent()
             $this->context->smarty->assign('wrong_install_name', true);
         }
 
-        if (basename(_PS_ADMIN_DIR_) == 'admin' && file_exists(_PS_ADMIN_DIR_ . '/../admin/')) {
+        if (
+            // The install is well finished
+            !file_exists(_PS_ROOT_DIR_ . '/var/.install.prestashop')
```

Comment:
Just want to make sure I get it all right (I don't yet fully understand usecase/purpose of this).

so if install failed,
the lock file will still exist and then
the `admin` dir won't be renamed.
So what happens next? it redirects somewhere, but it doesn't show any errors or anything like that about the unfinished install right? (so from UX point there are no changes).

So we just leave the lock file for module hooks (or basically for developers) to check its existence when needs to determine if install finished? (but when is it needed?, could you provide a brief example usecase?, I didn't get it from PR description fully)

Reply:
yes, sorry, but I didn't really get it fully when/who accesses the shop during the installation (therefore I wanted to get the example of a usecase, so Im sure we are on the same page).

Reply:
Who ? The Shop Administrator
When ? During the install
What ? The SA wants to access to the directory `admin` during the install

Reply:
okay, sorry, but then shouldn't it be handled somewhere like, some error shown that "installation is still in progress" in that case?

Reply:
It's the part to determine if install is complete. That avoid some errors when reaching "/admin" during installation (but not all).

Reply:
okay then